File: src/JStest.java
import jsjavax.swing.JApplet;
import jsjavax.swing.JLabel;


public class JStest extends JApplet {


	public void init() {
		JLabel j = new JLabel("Hello world!");
		add(j);
		System.out.println(getParameter("width"));
	}
	
	public static void main(String[] args) {

		JStest app = new JStest();
		app.setStub(new jsContext(app));
		app.init();
		int[][] a =  new int[3][4];
		
		int[][] b = new int[][] {new int[]{4,5},new int[]{5,6}}; 
		System.out.println(args);
  }
}

File: site/swingjs/j2s/JStest.js
Clazz.load (["jsjavax.swing.JApplet"], "JStest", ["jsContext", "jsjavax.swing.JLabel"], function () {
c$ = Clazz.declareType (null, "JStest", jsjavax.swing.JApplet);
Clazz.overrideMethod (c$, "init", 
function () {
var j =  new jsjavax.swing.JLabel ("Hello world!");
this.add (j);
System.out.println (this.getParameter ("width"));
});
c$.main = Clazz.defineMethod (c$, "main", 
function (args) {
var app =  new JStest ();
app.setStub ( new jsContext (app));
app.init ();
var a =  Clazz.newIntArray (3, 4, 0);
var b =  Clazz.newArray (-1, [ Clazz.newIntArray (-1, [4, 5]),  Clazz.newIntArray (-1, [5, 6])]);
System.out.println (args);
}, "~A");
});
